README.md

```diff
@@ -1,7 +1,150 @@
 
-This plugin provides `ZGW` functionality for the XXLLNC API. For More information on ZaakGerichtWerken and XXLNC, please visit [here]([https://github.com/vrijBRP/vrijBRP]\(https://xxllnc.nl/zaakgericht\)https://xxllnc.nl/zaakgericht)
+### Bundles Used in the XxllncZGWBundle
+
+The [**XxllncZGWBundle**](https://github.com/CommonGateway/XxllncZGWBundle) currently utilizes the following bundles:
+
+1.  **CoreBundle**: [GitHub Repository](https://github.com/CommonGateway/CoreBundle)
+2.  **ZGWBundle**: [GitHub Repository](https://github.com/CommonGateway/BRPBundle) (the ZGWBundle also includes other bundles like the KlantenBundle which is part of ZGW.)
+
+While the ZGW and CoreBundle can still be installed as standalone components (please refer to their respective installation guides), the XxllncZGWBundle now defaults to installing these bundles as additional plugins on the same gateway.
+
+## Backend Installation Instructions
+
+The XxllncZGWBundle backend codebase utilizes the Common Gateway as an open-source installation framework. This means that the XxllncZGWBundle library, in its core form, functions as a plugin on this Framework. To learn more about the Common Gateway, you can refer to the documentation [here](https://commongateway.readthedocs.io/en/latest/).
+
+The CommonGateway Admin UI frontend comes with te CommonGateway.
+
+To install the backend, follow the steps below:
+
+### Gateway Installation
+
+1.  If you do not have the Common Gateway installed, you can follow the installation guide provided [here](https://github.com/ConductionNL/commonground-gateway#readme). The Common Gateway installation is required for the backend setup. You can choose any installation method for the gateway, such as Haven, Kubernetes, Linux, or Azure, and any database option like MySQL, PostgreSQL, Oracle, or MsSQL. The gateway framework handles this abstraction.
+
+### XxllncZGWBundle Installation - Admin-UI
+
+1.  After successfully installing the Gateway, access the admin-ui and log in.
+2.  In the left menu, navigate to "Plugins" to view a list of installed plugins. If you don't find the "common-gateway/xxllnc-zgw-bundle" plugin listed here, you can search for it by clicking on "Search" in the upper-right corner and typing "xxllnc" in the search bar.
+3.  Click on the "common-gateway/xxllnc-zgw-bundle" card and then click on the "Install" button to install the plugin.
+4.  The admin-ui allows you to install, upgrade, or remove bundles. However, to load all the required data (schemas, endpoints, sources), you need to execute the initialization command in a php terminal.
+
+### XxllncZGWBundle Installation - Terminal
+
+1.  Open a php terminal and run the following command to install the XxllncZGWBundle:
+
+        docker-compose exec php composer require common-gateway/xxllnc-zgw-bundle
+
+### Initialization Command (Terminal)
```

Comment:
This is also fine,
but I do wonder if we mention this step in the CoreBundle -> Plugin.md docu as well and if we should repeat this here, probably yes.